src/blockdata/script.rs

```diff
@@ -289,11 +289,14 @@ impl Script {
         // push opcode (for 0 to 16) followed by a data push between 2 and 40 bytes gets a new
         // special meaning. The value of the first push is called the "version byte". The following
         // byte vector pushed is called the "witness program".
+        let mut ver = self.0[0];
+        if ver > 0 {
+            ver -= 0x50;
```

Comment:
This can cause a panic if it's given a value between 1 and 0x50.

Reply:
addressed